Source/WTF/wtf/ListHashSet.h

```diff
@@ -189,7 +189,7 @@ template<typename ValueArg, typename HashArg = DefaultHash<ValueArg>> class List
     iterator makeIterator(Node*);
     const_iterator makeConstIterator(Node*) const;
 
-    HashTable<Node*, Node*, IdentityExtractor, NodeHash, NodeTraits, NodeTraits> m_impl;
+    HashTable<Node*, Node*, IdentityExtractor, NodeHash, NodeTraits, NodeTraits, ShouldValidateKey::Yes> m_impl;
```

Comment:
I suppose eventually we’ll want `ShouldValidateKey::Yes` to be the default for `HashTable` or have no default.

Reply:
Yes, I thought the same thing. I'll follow up and do that after this batch of PRs lands.